setup.py

```diff
@@ -47,7 +47,7 @@ def package_files(directory):
 
 
 install_requires = [
-    'yara-python-dex>=1.0.1',
+    "yara-python-dex @ git+https://github.com/MobSF/yara-python-dex.git",
```

Comment:
> Wouldn't it be a risky change?

Yes, it could be that is why I was hesitant to raise the merge request, but there's no solution instead of it on termux(android), even if we go with toml setup we'll have to add `git+https://github.com/MobSF/yara-python-dex.git` in requirements for it to be able to complete installation successfully.

Or how about adding a note for It on README?

Reply:
Why are we pointing folks at a prebuilt thing that isn't controlled from the source we maintain? That feels bad.

I know I haven't been a good maintainer and been looking at this for a while, however if there are changes in the branched thing you're pointing at, why not just make a PR to the actual branch we control?